karols-frontend/src/containers/DatePickerContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react"
+import InfiniteCalendar from "react-infinite-calendar"
+import "react-infinite-calendar/styles.css" // Make sure to import the default stylesheet
+
+// Render the Calendar
+let today = new Date()
+let lastWeek = new Date(
+  today.getFullYear(),
+  today.getMonth(),
+  today.getDate() - 7
+)
+
+class DatePickerSelect extends Component {
+  state = {
+    date: new Date()
+  }
+
+  onChange = date => this.setState({ date })
```

Comment:
these are not used

Reply:
you should have an handleSelect Method (bind it in constructor) that will setState